cmd/osctl/cmd/list.go

```diff
@@ -23,45 +24,38 @@ import (
 
 const sixMonths = 6 * time.Hour * 24 * 30
 
+var (
+	long           bool
```

Comment:
reworked these flags to use variables to align other commands.

would like to push those global variables to be small internal structures passed to the command in the next round.